src/Core/Domain/Product/Combination/QueryResult/EditableCombinationForListing.php

```diff
@@ -70,6 +70,11 @@ class EditableCombinationForListing
      */
     private $quantity;
 
+    /**
+     * @var string|null
+     */
+    private $relativeImagePath;
```

Comment:
```suggestion
    private $imageUrl;
```

Since it will now contain the image URL we should rename it into a more suitable name

Reply:
haha I've done it before seeing your comment. Going the right way :sweat_smile: